Sniff.php

```diff
@@ -102,6 +104,21 @@ private function getTestVersion()
                     $arrTestVersions[$testVersion] = array($matches[1], $matches[2]);
                 }
             }
+            elseif (preg_match('/^\d+\.\d+-$/', $testVersion)) {
```

Comment:
I just copied the style of the existing code, which was already using three separate checks.  I am wary of building overly complex regexes which become very hard to maintain.  Probably the whole thing should be rewritten to use ```explode()```` instead, avoiding regexes altogether.